monkey/agent_plugins/exploiters/wmi/config-schema.json

```diff
@@ -7,6 +7,14 @@
             "minimum": 0,
             "default": 30.0,
             "maximum": 100.0
+        },
+        "smb_connect_timeout": {
+            "title": "SMB connection timeout",
+            "description": "The maximum time (in seconds) to wait for a response on an SMB connection.",
+            "type": "number",
+            "minimum": 0,
+            "default": 15.0,
+            "maximum": 100.0
```

Comment:
I don't think the user needs access to this. It's also confusing: The user may not know why any SMB settings are required on a WMI module.